content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -60,6 +60,96 @@ for a number of reasons:
   without many constraints and have namespaced names, such configuration is
   usually portable.
 
+## Bound service account tokens
+
+ServiceAccount tokens can be bound to API objects that exist in the kube-apiserver.
+This can be used to tie the validity of a token to the existence of another API object.
+Supported object types are as follows:
+
+* Pod (used for projected volume mounts, see below)
+* Secret (can be used to allow revoking a token by deleting the Secret)
+* Node (beta in v1.30)
+
+When a token is bound to an object, the object's `metadata.name` and `metadata.uid` are
+stored as extra 'private claims' in the issued JWT.
+
+When a bound token is presented to the kube-apiserver, the service account authenticator
+will extract and verify these claims.
+If the referenced object no longer exists (or its `metadata.uid` does not match),
+the request will not be authenticated.
+
+### Additional metadata in Pod bound tokens
+
+{{< feature-state feature_gate_name="ServiceAccountTokenPodNodeInfo" >}}
```

Comment:
does this automatically pull the beta status, or do we have to indicate that? all the other examples I see have to put the version and state in: https://grep.app/search?q=%7B%7B%3C%20feature-state&filter[repo][0]=kubernetes/website&filter[path][0]=content/en/

Reply:
The current proposed change looks good. It leverages the new `feature_gate_name` option which automates the feature status retrieval based on the information in the feature description file i.e. '[service-account-token-pod-node-info.md](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/command-line-tools-reference/feature-gates/service-account-token-pod-node-info.md?plain=1)'.

Reply:
(see https://kubernetes.io/docs/contribute/new-content/new-features/)